app/models/profile.rb

```diff
@@ -34,7 +21,23 @@ def activate
     end
   end
 
-  def verified?
-    verified_at.present?
+  def plain_pii
+    @_plain_pii ||= Pii::Attributes.new
```

Comment:
`Profile.encrypt_pii` also fingerprints the ssn, which must be done in sync with whatever PII is being encrypted. So having a method to handle all the encryption-related stuff seems ok to me.